### PR TITLE
Css three column layout - Dylan Dao

### PR DESCRIPTION
**Pull request by dilpills**

### CODE REVIEW

**Review by shawnkost — CHANGES_REQUESTED** [review]
Please include a screenshot of your page

**Comment by dilpills** [comment]
Screenshot 2023-03-30 at 3 58 05 PM

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <p id="motto">Designed for life science. Optimized for SaMD and Digital Health.</p>
+    </div>
+    <div class="row">
+      <div class="column">
+        <div class="imagediv"><img src="images/merge.png"></div>
```
Comment (shawnkost) [inline css-three-column-layout/index.html:19]: Missing an `alt` attribute

File: css-three-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="minihead">Streamlined QMS</div>
+        <div class="texts">
+          <div class="text">A QMS that includes modules for all stages of the product life cycle from Design Control to CAPA</div>
+        </div>
+      </div>
+      <div class="column">
+        <div class="imagediv"><img src="images/cloud-computing.png"></div>
```
Comment (shawnkost) [inline css-three-column-layout/index.html:26]: Missing an `alt` attribute

File: css-three-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="minihead">Integrates with existing tools</div>
+        <div class="texts">
+          <div class="text">Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</div>
+        </div>
+      </div>
+      <div class="column">
+        <div class="imagediv"><img src="images/discussion.png"></div>
```
Comment (shawnkost) [inline css-three-column-layout/index.html:33]: Missing an `alt` attribute

File: css-three-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  display: flex;
+  margin-bottom: 1rem;
+}
+
+.container,
+.column,
+.row {
+  display: flex;
+  align-items: center;
+}
```
Comment (shawnkost) [inline css-three-column-layout/styles.css:29]: `container` shouldn't have `display: flex`. Only `columns` or `rows`

File: css-three-column-layout/styles.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+.column,
+.container {
+  flex-direction: column;
+}
```
Comment (shawnkost) [inline css-three-column-layout/styles.css:34]: See note above about `container` ^

File: css-three-column-layout/styles.css [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+img {
+  width: 100%;
+  scale: 50%;
```
Comment (shawnkost) [inline css-three-column-layout/styles.css:38]: Does this actually do anything? I've never seen `scale` used like this. It's usually coupled with a `transform: scale()`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested Dylan!

File: css-three-column-layout/styles.css (hunk 6 above)
Comment (dilpills) [reply css-three-column-layout/styles.css:38]: Screenshot 2023-03-30 at 4 31 35 PM

Without scale, it worked so I just went with it.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**